easybuild/easyconfigs/j/JupyterLab/JupyterLab-4.0.5-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,73 @@
+easyblock = 'PythonBundle'
+
+name = 'JupyterLab'
+version = '4.0.5'
+
+homepage = 'https://jupyter.org/'
+description = """JupyterLab is the next-generation user interface for Project Jupyter offering all the familiar
+ building blocks of the classic Jupyter Notebook (notebook, terminal, text editor, file browser, rich outputs,
+ etc.) in a flexible and powerful user interface. JupyterLab will eventually replace the classic Jupyter
+ Notebook."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+builddependencies = [
+    ('binutils', '2.40'),
```

Comment:
@casparvl just checking that `maturin` was deliberately removed.

Reply:
Yes, I moved a few extensions from `JupyterLab` to `jupyter-server`. If I remember correctly, I did this because they were also needed by another extension in `jupyter-server`, the `pip check` was failing otherwise. `maturin` was needed by one of those (can't remember exactly which) extensions. Thus, the `maturin` build dependency also got moved to `jupyter-server`.

Reply:
Thanks for confirming.